files/zh-cn/glossary/w3c/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: W3C
+slug: Glossary/W3C
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
+---
+
+{{GlossarySidebar}}
+
+_万维网联盟_（World Wide Web Consortium）是一个维护 {{Glossary("World Wide Web", "万维网")}} 规则和框架的国际组织。
+
+万维网联盟（W3C）由超过 350 个成员组织组成，它们共同开发 Web 标准、运行推广计划并维护一个讨论 Web 的开放论坛。W3C 协调行业公司确保它们实施相同的 W3C 标准。
+
+每个标准都经过四个成熟度阶段：工作草案（WD）、候选推荐标准（CR）、建议推荐标准（PR）和 W3C 推荐标准（REC）。
```

Comment:
https://www.w3.org/zh-hans/news/2024/rdf-dataset-canonicalization-is-a-w3c-proposed-recommendation/

```suggestion
每个标准都经过四个成熟度阶段：工作草案（WD）、候选推荐标准（CR）、提案推荐标准（PR）和 W3C 推荐标准（REC）。
```